site/src/pages/SetupPage/SetupPageView.tsx

```diff
@@ -161,6 +210,112 @@ export const SetupPageView: FC<SetupPageViewProps> = ({
             </div>
           </label>
 
+          {form.values.trial && (
+            <>
+              <Stack spacing={1.5} direction="row">
+                <TextField
+                  {...getFieldHelpers("trial_info.first_name")}
+                  id="trial_info.first_name"
+                  name="trial_info.first_name"
+                  fullWidth
+                  label={Language.firstNameLabel}
+                />
+                <TextField
+                  {...getFieldHelpers("trial_info.last_name")}
+                  id="trial_info.last_name"
+                  name="trial_info.last_name"
+                  fullWidth
+                  label={Language.lastNameLabel}
+                />
```

Comment:
I unfortunately don't really have a choice re: the single `Name` input, since this is the info they need in Hubspot. Can probably be revisited when this is refactored into a new page.